charts/rstudio-pm/values.yaml

```
@@ -228,7 +228,7 @@ serviceMonitor:
 # -- config is a nested map of maps that generates the rstudio-pm.gcfg file
 config:
   Server:
-    RVersion: "/opt/R/3.6.2/"
+    RVersion: "/opt/R/4.2.0/bin/R"
```

Use the default version (symlink) inside the container.
```suggestion
    RVersion: "/opt/R/default/"
```